log4j-jul/pom.xml

```diff
@@ -29,56 +29,58 @@
   <description>The Apache Log4j implementation of java.util.logging</description>
 
   <properties>
-    <log4jParentDir>${basedir}/..</log4jParentDir>
-
-    <!--
-      ~ OSGi and JPMS options
-      -->
     <bnd-extra-package-options>
-      <!-- Log4j Core is optional -->
-      org.apache.logging.log4j.core.*;resolution:=optional
+      <!-- Optional annotations -->
+      org.jspecify.*;resolution:=optional
     </bnd-extra-package-options>
-
-    <!-- PTS requires using the JUnit Platform, which interferes with JUL initialization -->
-    <predictive.test.selection.enabled>false</predictive.test.selection.enabled>
+    <bnd-extra-module-options>
+      <!-- Optional modules can not be `transitive` -->
+      org.jspecify;transitive=false
+    </bnd-extra-module-options>
   </properties>
 
   <dependencies>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-api</artifactId>
     </dependency>
+
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-core</artifactId>
-      <optional>true</optional>
+      <artifactId>log4j-kit</artifactId>
```

Comment:
I was expecting only a dependency on `log4j-api`. Can we do without `-kit`?

Reply:
The `log4j-jul` modules use to Log4j configuration properties, `log4j-kit` is necessary if we want to support them in a 3.x-way.